LICENSE.md

```diff
@@ -1,5 +1,7 @@
 Copyright (c) 2022 MapLibre contributors
 
+Copyright (c) 2017 Mapbox Inc.
```

Comment:
Was this code originally copyrighted/licensed by Mapbox?  Why was it missing in the first place?  Adding / Removing should be a question for the Maplibre board.

Reply:
Have a look at the history: https://github.com/maplibre/maplibre-react-native/commits/main/LICENSE.md

This project was started by Mapbox. The copyright notice was removed roughly around the point the project went into community hands. But still the copyright notice was around in many file headers. So by removing the individual file headers, I would say we do right by adding/keeping the copyright in the LICENSE.

We even may drop the last paragraph:

> This project was originally forked from https://github.com/rnmapbox/maps, also under the MIT license, but without a copyright holder notice.

What do you think @lseelenbinder?

Reply:
Let's make it Copyright 2017-<year it was forked>, but otherwise I think this is okay to align with Native. The worst case here is we replace the notification, but regardless let's make sure it stays in the license.